src/shared/components/Checkbox/Checkbox.styles.tsx

```diff
@@ -0,0 +1,90 @@
+import theme from '@/shared/theme'
```

Comment:
```suggestion
import { sizes, colors } from '@/shared/theme'
```
This is just for consistency, we import in this manner everywhere in the application